README.md

```diff
@@ -1,7 +1,7 @@
 # Summary
 
 This toolkit provides methods to execute the TPC-H, TPC-DS, and SSB benchmark on
-Swarm64 DA and native PostgreSQL.
+Swarm64 DA, native PostgreSQL and EDB Postgres Advance Server (EPAS).
```

Comment:
Two minor things:
* "Advanced Server"
* And maybe consider to write "PostgreSQL" and possibly "PostgreSQL with Swarm64 DA"

Reply:
Done